data/adapters/adastra_asb_adapter.py

```diff
@@ -0,0 +1,95 @@
+import csv
+from adapters import Adapter
+from adapters.helpers import build_variant_id
+
+# Example ADASTRA allele-specific binding (ASB) file TF_name@cell_name.tsv (e.g. ATF1_HUMAN@HepG2__hepatoblastoma_.tsv):
+# chr	pos	ID	ref	alt	repeat_type	mean_BAD	mean_SNP_per_segment	total_cover	n_aggregated	es_mean_ref	es_mean_altlogitp_ref	fdrp_bh_ref	logitp_alt	fdrp_bh_alt	motif_log_pref	motif_log_palt	motif_fc	motif_pos	motif_orient	motif_conc	novel
+# chr11	129321262.0 rs10750410	A	G		1.25	518.5	73.0	2.0	-1.508200583122832	1.5220631227173078	0.9999438590195968	1.0	3.776801544248756e-06	0.0024711390339222	2.668977799202377	2.9239362481887974	0.8469536347168959	19	+	No Hit	False
+
+
+class ASB(Adapter):
+    # 1-based coordinate system
+
+    DATASET = 'asb'
+
+    def __init__(self, filepath, TF_ids, cell_ontologies):
+        self.filepath = filepath
+        self.TF_ids = TF_ids
+        self.cell_ontologies = cell_ontologies
+        self.dataset = ASB.DATASET
+
+        super(ASB, self).__init__()
+
+    def get_TF_uniprot_id(self):
+        TF_name = self.filepath.split('/')[-1].split('@')[0]
```

Comment:
in python, variables that start with a capital letter are interpreted as constants, we should rename it to `tf_name` for example (even though it all works fine :P)